test/cartesian/Area.animation.spec.tsx

```diff
@@ -27,55 +27,106 @@ function getAreaCurveD(container: Element): string {
 
 const expectedUvLabels: ReadonlyArray<ExpectedLabel> = [
   {
-    height: null,
```

Comment:
width and height don't do anything in `text` SVG element, we should remove them.